### PR TITLE
opsys: Remove rhel.py file

### PR DESCRIPTION
**Pull request by mkutlak**
The file is missing implementation and it isn't needed anymore.

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

File: faf.spec.in [hunk 1]
```diff
@@ -816,9 +816,6 @@ fi
 %config(noreplace) %{_sysconfdir}/faf/plugins/fedora.conf
 %{python_sitelib}/pyfaf/opsys/fedora.py*
 
-%files opsys-rhel
-%{python_sitelib}/pyfaf/opsys/rhel.py*
```
Comment (xsuchy) [inline faf.spec.in]: As this is only one file in subpackage opsys-rhel, this can be safely removed (see top of spec file)

**Comment by marusak** [comment]
add to whitelist

**Comment by marusak** [comment]
ok to test

**Comment by marusak** [comment]
test this please

**Comment by marusak** [comment]
add to whitelist

**Comment by marusak** [comment]
add to whitelist

**Pull request merged**